daemon/parser.py

```diff
@@ -26,7 +26,7 @@ def mixin_daemon_parser(parser):
     gp.add_argument(
         '--store',
         action='store_true',
-        default=False,
+        default=True,
```

Comment:
using `store_true` action with `default=True` is basically making `store` as a `True` constant

Reply:
yeah maybe should actually remove this option. Not sure we still need it.